### PR TITLE
LPD-38758 translate shows the duplicate fields - playwright test

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,
May I ask to review this PR .

Is it for https://liferay.atlassian.net/browse/LPD-38758. 

Solution: created TranslationsInterface#CanViewDuplicateFields test in playwright.

The issue was with the button to create the duplicated field. It is working in PW if we click outside first.

Removed the Poshi test as well.

Thank you!

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Comment by adolfopa** [comment]
ci:reevaluate:1397401_11454

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 1]
```diff
@@ -637,6 +637,60 @@ baseTest(
...
+
+		await apiHelpers.dataEngine.createStructure(site.id, dataDefinition);
+
+		await journalEditArticlePage.goto({
+			siteUrl: site.friendlyUrlPath,
+			structureName,
+		});
+		const title = getRandomString();
+
+		await journalEditArticlePage.titleInput.fill(title);
```
Comment (ambrinchaudhary) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:663]: I think `fillAndClickOutside` will reduce possible flakiness, just like you did in https://github.com/liferay-content-management/liferay-portal/pull/5965/files

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 2]
```diff
@@ -637,6 +637,60 @@ baseTest(
...
+		await journalEditArticlePage.duplicateButton.click();
+
+		await page
+			.locator(
+				'input[id^="_com_liferay_journal_web_portlet_JournalPortlet_ddm$$Text"]'
+			)
+			.nth(1)
+			.fill('Duplicated Text Field');
+
+		await journalEditArticlePage.publishButton.click();
```
Comment (ambrinchaudhary) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:680]: I would move all this above steps (from L657) to a function in JournalEditArticlePage, something like `createArticleWithDuplicateField` 🤔 . So the test is more clear.

File: modules/test/playwright/tests/journal-web/journalArticle.spec.ts [hunk 3]
```diff
@@ -637,6 +637,60 @@ baseTest(
...
+		await journalPage.goToJournalArticleAction('Translate', title);
+
+		const duplicateFields = page.locator(
+			'[id^="_com_liferay_translation_web_internal_portlet_TranslationPortlet_infoField--DDMStructure_Text"]'
+		);
+
+		await duplicateFields.first().waitFor({state: 'visible'});
+
+		const count = await duplicateFields.count();
+		expect(count).toBe(2);
```
Comment (ambrinchaudhary) [inline modules/test/playwright/tests/journal-web/journalArticle.spec.ts:691]: Just `await expect(duplicateFields.count()).toBe(2)`.

**Review by ambrinchaudhary — CHANGES_REQUESTED** [review]

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:content-management-playwright

**Pull request closed without merging**